.github/workflows/comment.yaml

```diff
@@ -13,57 +13,44 @@ env:
   ARTIFACT_NAME: "report"
 
 jobs:
-  download-artifacts:
-    name: Download Artifacts
+  filter-artifacts:
+    name: Filter Artifacts
     runs-on: ubuntu-24.04
     if: github.event.workflow_run.event == 'pull_request'
     outputs:
-      artifacts: ${{ fromJson(steps.download-artifact.outputs.result) }}
+      artifacts: ${{ fromJson(steps.filter-artifacts.outputs.result) }}
     steps:
       - uses: actions/checkout@v4.2.2
-      - id:   download-artifact
-        name: Download test report artifact
+      - id:   filter-artifacts
+        name: Filter test report artifacts
         uses: actions/github-script@v7.0.1
         with:
           script: |
             const fs = require('fs');
-            const owner = context.payload.workflow_run.repository.owner.login;
-            const repo = context.payload.workflow_run.repository.name;
-            const run_id = ${{ github.event.workflow_run.id }};
             const artifacts = await github.rest.actions.listWorkflowRunArtifacts({
-                owner: owner,
-                repo: repo,
-                run_id: run_id,
+                owner: context.payload.workflow_run.repository.owner.login,
+                repo: context.payload.workflow_run.repository.name,
+                run_id: ${{ github.event.workflow_run.id }},
             });
-            const report_artifacts = artifacts.data.artifacts.filter(artifact =>
+            const zips = artifacts.data.artifacts.filter(artifact =>
               artifact.name.startsWith('${{ env.ARTIFACT_NAME }}-')
-            )
```

Comment:
I think this will make the output a list of artifact objects instead of a list of artifact names, which I suppose is the intended input for the `comment-on-prs` job?